test/sources.js

```diff
@@ -355,38 +355,120 @@ describe('Sources', function() {
   });
 
   describe('Health', function() {
-    const stubs = setUp();
+    this.timeout(5000);
```

Comment:
We should avoid setting test timeouts as tweaking this can mask actual timing issues in our code.

Reply:
will look

Reply:
As mentioned, the previous section of the tests (`initialization`) already has a timeout value, which indicates this is something to do with the initialization process, which has always been in